package.json

```diff
@@ -1,26 +1,49 @@
 {
     "name": "pact-gen-ts",
     "version": "0.0.1",
+    "description": "Generating pact files from typescript definitions",
+    "keywords": [
+        "pacts",
+        "contracts",
+        "tests",
+        "typescript"
+    ],
     "main": "./lib/index.js",
+    "bin": {
+        "pact-gen-ts": "./bin/pact-gen-ts.js"
```

Comment:
Maybe "pact-gen" as the cli invocation would be enough? Its always a bit less typing :)

Reply:
I think that the ending "ts" is very important for command which generating pacts from typescript.